### ISSUE TITLE
[0.7.4.0 staging] Typo in Mods\AutoGen\Food\Flour.cs

### ISSUE THREAD

**Issue by Thineboot** [StrangeLoopGames/EcoIssues#7631]
**E** in Description  
```
Line 20:
    public partial class FlourItem : FoodItem            
    {
        public override string FriendlyName                     { get { return "Flour"; } }
        public override string FriendlyNamePlural               { get { return "Flour"; } } 
        public override string Description                      { get { return "A fine, milled wheat product thatE's useful for all baking."; } }

        private static Nutrients nutrition = new Nutrients()    { Carbs = 15, Fat = 0, Protein = 0, Vitamins = 0};
        public override float Calories                          { get { return 50; } }
        public override Nutrients Nutrition                     { get { return nutrition; } }
    }
```

**Comment by Thineboot** [comment]
Fixed it 0.7.4.0

**Comment by craig-jong** [comment]
Does that mean its fixed? I don't see it in either the XLSM or the CSV. I'm going to close this, but I guess its possible the generator is doing something weird on the build side.
